forge-gui/res/cardsfolder/upcoming/the_master_formed_anew.txt

```diff
@@ -0,0 +1,9 @@
+Name:The Master, Formed Anew
+ManaCost:U B
+Types:Legendary Creature Time Lord Rogue
+PT:0/1
+K:ETBReplacement:Copy:DBCopy:Optional
+T:Mode$ SpellCast | ValidCard$ Card.Self | Execute$ TrigExile | OptionalDecider$ You | TriggerDescription$ Body Thief — When you cast this spell, you may exile a creature you control and put a takeover counter on it.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | Hidden$ True | ChangeType$ Creature.YouCtrl | ExileWithCounter$ TAKEOVER

```

Comment:
This is going to require a new type of counter.

Reply:
check recent commits to `CounterEnumType.java` for how it works
color is whatever you feel fits